### ISSUE TITLE
Aura condition - "Any of" ui glitches

### ISSUE THREAD

**Issue by shighman** [WeakAuras/WeakAuras2#3791]
### Description

When creating an always on (either cd or buff) and then going into conditions and using the "Any of" option with more than 2 conditions, when creating it it looks fine but on later review all of the edit boxes on the right glitch and don't show. 

Attached screenshot, this occurs whether Weak Auras is the only available addon or not, and happens with both progress bar and icon weak auras. Any spell not just the spell listed in the steps below. 

It seems somewhat intermittent but shows incorrectly much more than not.

### WeakAuras Version

WeakAuras 4.1.2

### World of Warcraft Flavor

Retail (Default)

### Tested with only WeakAuras

- [X] Yes
- [ ] No

### Reproduction Steps

Create a new progress bar weak aura
Set trigger to 
  Type: Aura
  Unit: Player
  Aura type: Buff
  Names: Bone Shield
  Show On: Always

Go to conditions tab, add condition:
  If: Any Of
  Trigger 1: Remaining Duration < 2
  Trigger 2: Aura(s) found = false
  Then: Bar color: Red
  And: Sound, Play, Custom, master, <insert sound file>

Add a second condition, remaining duration >= 2
Then bar color: green.

Close weak auras, go back into options and navigate to the conditions tab. Occasionally it will render correctly, but most of the time it will appear like the screenshot. If you click onto different weak auras and then go back sometimes it might work. 

### Screenshots

![image](https://user-images.githubusercontent.com/10523799/187729267-4ade0a75-60fe-4955-b551-6cb4dae599f6.png)

### Export String

!WA:2!1v1ZVnXXx8SDLaHvBf4wIAqusAujcqL0GjekQarEtSXbCS931leGIK9S7m27qwV7QzM1jo3WQhqTQOwFO)b4Z9Kp0)aI61knYQQN)Q8Nq(lOVzwRear2d2Z8(18MpV37ZyumBNS4S4xV0qEmIT9wuSW)87RxxTvlor8uJXOeHFeRASGgfYZW6LQO8PhHc9af1IOHc31kuXPG9yTN2rcKYyJ4Z9o73mctSY)yNQ5sybF0HUiVTBZIscXRffeXEOHHrnZLw8w9fmA72eg)txGnz5)Ys4KkOoKMdqjmuiSIFHrwrHK5Q7tjb4XDqcpFcVUF0ovdhYH)YhSdQh3M0LaPxjckq4BLesfUXbOEeMToiz2xfpEmjiGI5xW9MRSYTVZY91c2aZZCapXvhHAmsl6U21RvOC5JewpPfiCuJ1Yx3PrDN82owIEXeBvmZDiKYf2f5jQNgm)HyIl4GdybRuHY1k(4YdscNCdZeFoWuAxIt6Efw9htnK6ff2WtdpMW3HautyHOGNaWdGOBpGtcAPRaYt13fXsrstddt404ibKicI)BxqFMXqi9TjTvvZhoTC2XW2Jlfte0kIrsf86zIZki7kAW9r4ODEAAumhOLP(z(lhhF23YIJkMMhKk9Om8mXFMwc0qfb1lQNUDZkpS9qTcamjyTWxnoT56zPhNX(A9VmHlOT6jp1qMQLI4acTQuTsH45okDAaPoe8gXne0oKgDIWnuyqt5Sm4EPSysONCtmgQ96ymEWKOekIpBrgDV5(Fjimec0CooXF1jDoXmIhvvsmFh4As(FXPIN5d4z6cBvaWJ1Q2jIH3IHI7V1KfPx8UqODdinNa0V1G3WnQuPGDJYfk6eFPtk5s3zm(OBMQh0Qcm(CSO609iZeF1tke4EW8c1RHWNrGPRaCV4V(KSnG0g51RrRGOiwZxFE53A(bU98jlScH0qU08xMlVHURrMtEg5nLllV1RKRyiVT8uYVtEh5SYVxEpDbsEF5QY8xCk56YcYInLpyukgyVXdk5iFO8rYYZiVRXh4mdNSqwZ1NqB7lM1kicH7oagvB4fG48MUcuamzpTDNKabntFLgGnWRPTw)cUCAy7aYO1lK3P0JQOpYTMw(dnTuwTG85MYTYmuhVOoUibiha2PbHhOhcTGOJvd35YV(6XNfmmpw1wtWe8MOD9hPTQefJjH2vk8Kc2dqH0oA(ZvS5cetuCFcItQlyKW2c)tditTXy1Goysdf)dJtaAdmVVYovPwwZQdIgwu2sBTST0xsbqOfnKY9FFXwksh)3p3OH(dzAwdnbMIId4By7ozkAGMPItXeBDDWf22I2otk1JAqnHrIVYgk(RwipYlwdbZYHTTqSx84nU(X7UEDLdtGbDr6EmrQ79TcO7ThIHZLqXJwUCUxIOpj8kcM1ErrDmS1(4pue51nLE8lmA2yEWu3JFOWEhf9Yp(X2OGyFK5HPJsfzWJbQ7LB91SluOYOwkbaedJ9MPpnU2reemAOk(eFn7C9OeMh5ItnkIrHghDvyyPQ2B88QvCYxEOQsq1pDETfSHNO82UO8t(MP63fXOiyME(Q2ZhfN9UUAD8RbAnL)8bKDJPPLuhyKsBHDxuqcjBUv0w4QElHGLVXK55JcBtG4lFdq9xA1)CA9Nz)yweuQe9KxfuvCixXNR7qSQbVbozFms4hp7XfM6kP8xOECnfYwSt8nT12YMZdAhI6mo1t15gscC3ekEeM83K)(cYFkn)L)cKYZF)7jFt2CYF9cQmR0Q)9xQ(UuPv))lP)kT6F956ptW3RoIgMoDc35mm1JHaMTIKKrUDglOrvKjl3E5fVXI5Y29FE6)9d

**Comment by InfusOnWoW** [comment]
I have tried your steps now several times and imported your export. I couldn't reproduce your screenshot.

Can you put a percentage to "It seems somewhat intermittent but shows incorrectly much more than not.", with all other addons disabled?

**Comment by shighman** [comment]
Yes all other addons were disabled and happened with or without all other addons.

**Comment by shighman** [comment]
Yes all other addons were disabled and happened with or without all other addons.

**Comment by InfusOnWoW** [comment]
And how often does it happen?

**Comment by shighman** [comment]
With this weak aura every time, the ui glitch is not the same each time, i do have a lot of weak auras. 

Here is a video of the current glitchy view, https://www.youtube.com/watch?v=qwHeHvkkyvE

The weak aura does work, and if i click around i can get it to render correctly sometimes.

**Comment by InfusOnWoW** [comment]
Well if you have other weakauras loaded, then your aren't doing your reproduction steps.

**Comment by InfusOnWoW** [comment]
Also you video is showing a clearly skinned version of WeakAuras.

**Comment by shighman** [comment]
I deleted all my addons removed it from the group and deleted all the loaded weak auras.

![image](https://user-images.githubusercontent.com/10523799/188245076-7c6258be-0664-4c89-94f8-88c7a5d16b6f.png)
![image](https://user-images.githubusercontent.com/10523799/188245089-8be96668-462e-4f8a-a6ce-adc831b75e97.png)

The weak aura actually works if you are unable to replicate it, then you can close the issue.

**Comment by mrbuds** [comment]
I can't reproduce the problem either.
On you last post there is a "ghost" background of a dropdown next to "bar color"

**Comment by InfusOnWoW** [comment]
Yes that's even stranger, there's a ghost combobox, but half of it is behind remaining duration, and half of it in front of the bar color.

We believe you that the bug is real. But we need to be able to reproduce it to fix it. Reproducing bugs requires removing as many variables as possible.

But I tried again for the probably 5th time, and no luck.

I guess we can do one final thing to maybe gather a bit more information. That is the next time you encounter the bug, record a video, enable "/fstack" and slowly move around where the ghosts are shown. I doubt that there is anything interesting visible, but maybe it shows something strange.

Closing for now, as I'm out of ideas on how we can further investigate this.
